content/docs/manage/accounts.md

```diff
@@ -40,3 +40,23 @@ See [Organizations](/docs/manage/organizations) to learn more.
 Easily switch between your personal account and any organization you are a Member of using the navigation breadcrumb.
 
 ![Switch between personal and organization](/docs/manage/switch_to_org.png 'no-border')
+
+## Deleting your account
+
+Before deleting your account, you must take deliberate steps to remove your resources. This includes deleting all projects, leaving any organizations, and, if you're on a paid plan, downgrading to the Free Plan.
+
+To protect your data, each resource must be removed individually &#8212; nothing is deleted in bulk. The **Delete** action will stay disabled until all conditions are met.
+
+![Alt text](/docs/manage/delete_account.png)
+
+Here's where to go in the Neon Console to complete these actions:
+
+| **Action**                  | **Instructions**                                                                                                                                                                                                                                                                         |
+| --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Leave all organizations** | On the **People** page of each organization you belong to, find your name and select **Leave organization** from the kebab. If you're the only Admin, you'll have to promote another Member to Admin first before you can leave. <br/> ![leave organization](/docs/manage/leave_org.png) |
```

Comment:
"kebab" — is this really a word we use in our docs? Sounds kinda silly compared to "triple dots", "action menu" or sth like that.

Also, "you'll have to promote another Member to Admin" — or delete the organization.

Reply:
Yeah, we use it (kebab) in a couple places. It's a standard term, but may be too inside-baseball. Will see if there's a better term and change it globally.

Reply:
Delete option is added.